### ISSUE TITLE
[Platform SRE] Establish single point of contact for support requests

### ISSUE THREAD

**Issue by jhouse-solvd** [department-of-veterans-affairs/va.gov-team#36124]
## Problem Statement 
Support is confusing when there are simultaneous conversations for different purposes. The request is separate from the investigation. Mixing the two is confusing, hard to follow, and even harder to track. 

### Specific pain points
- Lack of clear ownership of a customer support request
- Conversations that are hard to follow for the customer _and_ for support personnel 
- Lack of resolution from the customer perspective 

## How might we
...separate the request from the investigation? 
...establish clear ownership of communication with the customer? _ie "single point of contact"_
...establish clear ownership for resolution of the issue? 
...provide clear acknowledgment, updates, and next steps?
...avoid duplicative/confusing discussions? 
...promote knowledge transfer between SMEs and SREs? 
...reduce disruptions for platform personnel that are not on-call? 

## Hypothesis or Bet
This will improve the quality of support provided to VFS teams. 
There will be little to no change for VFS teams (they will request support in the same way that they do today).
This will reduce broadcasts and redundant conversations for platform support personnel. 

## We will know we're done when... ("Definition of Done")
*What requirements does this project need to meet for you to finish this initiative?*

## Known Blockers/Dependencies 
Participation from the SRE team and Infrastructure teams

## Projected Launch Date
Feb 2022 

### ... tested?

- [ ] Usability test (_TODO: link_) has been performed, to validate that new changes enable users to do what was intended and that these changes don't worsen quality elsewhere. If usability test isn't relevant for this change, document the reason for skipping it.
    - [ ] ... and issues discovered in usability testing have been addressed.
    * _Note on skipping: metrics that show the impact of before/after can be a substitute for usability testing._
- [ ] End-to-end [manual QA](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/README.md) or [UAT](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/planning/what-is-uat.md) is complete, to validate there are no high-severity issues before launching
- [ ] _(if applicable)_ New functionality has thorough, automated tests running in CI/CD

### ... documented?

- [ ] New documentation is written pursuant to our [documentation style guide](https://vfs.atlassian.net/wiki/spaces/AP/pages/622264362/Style+guide)
- [ ] Product is included in the [List of VSP Products](https://docs.google.com/spreadsheets/d/1Fn2lD419WE3sTZJtN2Ensrjqaz0jH3WvLaBtn812Wjo/edit#gid=0)
    * _List the existing product that this initiative fits within, or add a new product to this list._
- [ ] Internal-facing: there's a [Product Outline](https://vfs.atlassian.net/wiki/spaces/PMCP/pages/1924628490/Product+Outline+Template)
- [ ] External-facing: a [User Guide on Platform Website](https://vfs.atlassian.net/wiki/spaces/AP/pages/1477017691/Platform+website+guidelines) exists for this product/feature tool
- [ ] _(if applicable)_ Post to [#vsp-service-design](https://dsva.slack.com/channels/vsp-service-design) for external communication about this change (e.g. VSP Newsletter, customer-facing meetings)

### ... measurable

- [ ] _(if applicable)_ This change has clearly-defined success metrics, with instrumentation of those analytics where possible, or a reason documented for skipping it.
    * For help, see: [Analytics team](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Analytics-customer-support-guide.1586823275.html)
- [ ] This change has an accompanying [VSP Initiative Release Plan](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new/choose).

### When you're ready to launch...
- [ ] Conduct a [go/no-go] (https://vfs.atlassian.net/wiki/spaces/AP/pages/1670938648/Platform+Crew+Office+Hours#Go%2FNo-Go) when you're almost ready to launch. 

### Documentation
* **`PRODUCT_NAME`**: _directory name used for your product documentation_
* **Product Outline**: _link to Product Outline_
* **User Guide**: _link to User Guide_

### Testing

* **Usability test**: _link to GitHub issue, or provide reason for skipping_
* **Manual QA**: _link to GitHub issue or documented results_
* **Automated tests**: _link to tests, or "N/A"_

### Measurement

* **Success metrics**: _link to where success metrics are measured, or where they're defined (Product Outline is OK), or provide reason for skipping_
* **Release plan**: _link to Release Plan ticket_

## TODOs
- [ ] Convert this issue to an epic
- [ ] Add your team's label to this epic

**Comment by jhouse-solvd** [comment]
Riffing some thoughts...

One approach: 

- create an internal-facing channel for SRE to coordinate with SMEs from various platform teams
- dedicated SMEs from platform teams join the above channel during their on-call rotation 
- the SRE team asks for guidance and knowledge from SMEs in the above-mentioned Slack channel as needed
- the SRE team (only) communicates with the customer in #vfs-platform-suport

**Comment by zachclarity** [comment]
Living document:  Support Request process escalation - 
 https://vfs.atlassian.net/wiki/spaces/SRE/pages/2066546755/Support+Requests+-+Contacts+Escalation+Process

**Comment by little-oddball** [comment]
This is literally what we are working towards w/ some of the process and communication flow being put in place, particularly around tier 1 and tier 2 engagement.

**Comment by little-oddball** [comment]
OBE
